### DISCUSSION TITLE
prisma generate not updating the prisma client

### DISCUSSION THREAD

**Discussion by henriquesati** [prisma/prisma#16150] [Q&A]
I was tring to create a new collection using mongodb, but when I tried to run `npx prisma generate`, the Typescrit file was not recognizing the new collection on the query (that was supposed to be generated by the new client) and underlining the new collection name in red (as if it were a typo or non-existent variable): `const allPersons = await prisma.**person2**.findMany() `. After a few tries and searching, i'm being able to resolve just closing and reopening Visual studio code.

**Comment by nurul3101** [comment]
Hello @pedrogatinhos 👋 
In these cases, you can even just restart the typescript service, and it should update the generated types correctly.
There shouldn't be a need to reopen Visual studio code.
You can click `cmd+shift+p` and select `Restart TS server`
<img width="962" alt="Screenshot 2022-11-08 at 4 36 15 PM" src="https://user-images.githubusercontent.com/13049676/200548780-414b21c0-29b9-4931-bb57-4445889b614e.png">

**Reply by vladi-stevanovic** [reply]
👋 Hi @pedrogatinhos , did you perhaps have a chance to check Nurul's recommendation? Let us know if you still have any questions! 

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
